docs/upgrade.html

```diff
@@ -119,6 +119,13 @@ <h5><a id="upgrade_200_notable" href="#upgrade_200_notable">Notable changes in 2
     </li>
     <li><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-290%3A+Support+for+Prefixed+ACLs">KIP-290</a> adds the ability
         to define ACLs on prefixed resources, e.g. any topic starting with 'foo'.</li>
+    <li><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-283%3A+Efficient+Memory+Usage+for+Down-Conversion">KIP-283</a> improves message down-conversion
+        handling on Kafka broker, which has typically been a memory-intensive operation. The KIP adds new topic and broker configurations <code>message.downconversion.enable</code>
```

Comment:
I'd suggest moving the mention of the config to the end. The improvement in memory utilization is the core feature here.